common/app/common/ApplicationMode.scala

```diff
@@ -0,0 +1,19 @@
+package common
+
+import play.api.{Application, GlobalSettings}
+import play.api.Mode._
+
+object ApplicationMode {
+  private var theMode = Prod
```

Comment:
then we can stop it being set more than once too (although it's all getting a bit javay at that point!)